portal/site_persistence.py

```diff
@@ -382,6 +387,24 @@ def update_intervention(intervention_json):
                     "site_persistence: {}".format(repr(apptext)))
                 db.session.delete(apptext)
 
+        # ScheduledJobs shouldn't be order dependent, now is good.
+        jobs_seen = []
+        for j in objs_by_type['ScheduledJob']:
+            job = ScheduledJob.from_json(j)
+            db.session.add(job)
+            jobs_seen.append(j['name'])
+
+        # Delete any ScheduledJobs not named
+        if not keep_unmentioned:
+            jobs_seen.append("__test_celery__")
```

Comment:
Not sure I get the addition of `__test_celery__`.  Is that one you just want added in there, and expect would never be named in the persistence file?